tsgen/__main__.py

```diff
@@ -34,30 +34,34 @@ class TelemetrySystemGenerator:
     OUTPUT_FOLDER = './out'
     SENSORS_FILE_NAME = 'sensors.json'
     SCHEMA_FILE_NAME = 'schema.json'
-    C_DATABASE_NAME = 'can_database'
+    CAN_S_APP = 'can_s'
+    CAN_C_APP = 'can_c'
```

Comment:
I think rather than `_APP` these should be named like `CAN_C_DATABASE_NAME` since they are no longer appended to a common prefix.

Reply:
This is no longer relevant if we go with the idea of just generating the source file names from the name of the DBC file itself.